servicetest/filesystem/test_filesystem.py

```diff
@@ -41,18 +42,34 @@ def logfile_path():
     Container.HOST_PATH: CURRENT_DIR
 }
 
+TESTFILE='lala.txt'
```

Comment:
blah blah blah ;) Yes yes, you are correct. I had to run in the middle of things and wanted to commit what I had "for now"... I'll fix it tonight.